### ISSUE TITLE
Video stops playing after .detach() => appendTo with Chrome, Edge, but continues with Firefox.

### ISSUE THREAD

**Issue by dispeak** [jquery/jquery#3635]
### Description ###
I use the $.detach() function to move an html5 video from one DOM div to another.

If the video is currently playing:
1- tested with Firefox 53, the video continues playing after detach => appendTo the new element.
2- tested with Chrome 56 and 57, and Edge 38, the video stops playing after detach => appendTo to the new element.

I would expect 1 to be the correct behavior.

### Reduced test case ###
http://test.digitallyspeaking.com/playerJquery.html

Thank you for taking a look.

**Comment by dmethvin** [comment]
This is some browser-specific behavior as you've seen. Does it only occur when jQuery manipulates the page, or does it happen with standard DOM calls as well?

**Comment by dispeak** [comment]
Thank you Dave.

I have not been able to test that. I tried to look at the source code in jQuery, but it went above my head... I guess I am not sure how to do this, and at what level I should expect jQuery to abstract browser's behavior.

**Comment by dmethvin** [comment]
Should be something like this in plain DOM:
```js
var video = document.getElementById("myvideo");
var newplace = document.getElementById("newplace");
newplace.appendChild(video);
```

**Comment by dispeak** [comment]
Thanks for the guidance. It behaves exactly the same way with plain JS

http://test.digitallyspeaking.com/playerPlainJS.html

**Comment by dmethvin** [comment]
OK, you'd need to file a bug with the browsers that don't behave the way you want. Or perhaps you could work around the problem by forcing the video to start paying again once it was inserted. It's not something jQuery can do behind the scenes.

**Comment by dispeak** [comment]
Thanks for the clarification. Yes, I did add the workaround, and it works decently enough.
